shaders/fragment.glsl

```diff
@@ -1,4 +1,4 @@
-#version 330
+#version 130
```

Comment:
Yeah, as @Dr-Emann mentioned, I don't think this version is supported on newer Macs, which is why I bumped it up.